### DISCUSSION TITLE
看了文档不太理解，vfor的用法，比如列表渲染了一堆label这个例子能讲一下实现和原理吗，拜托，本人刚接触python不久

### DISCUSSION THREAD

**Discussion by tetrarrow** [CrystalWindSnake/ex4nicegui#205] [Q&A]
`for i in range(10):

    xx=rxui.label(i)
    xx.on('click',lambda:targ.set_text(xx.text))

targ=rxui.label('1')#我要怎么做才能，上面的label点击哪个这里显示哪个

ui.run()`

如果用普通的ui.label，这里只能相应最后一个也就是9。
麻烦作者大大，本人小白，可能问题很傻，抱歉

**Comment by CrystalWindSnake** [comment]
@tetrarrow 你这个例子与 vfor 无关。

```python
for i in range(10):
      ui.label(i).on('click',lambda i=i: ui.notify(i))
```
